keyboards/coban/pad3a/rules.mk

```diff
@@ -0,0 +1,4 @@
+# MCU name
+MCU = RP2040
+BOOTLOADER = rp2040
+ENCODER_ENABLE = yes
```

Comment:
If encoder feature is enabled at the keyboard level, you should have some sort of default encoder behaviour in `pad3a.c`, so it works for QMK Configurator users as well.

Reply:
Thanks for your suggestion, I have added default behaviour for encoder in `pad3a.c`.